paper/writeup.tex

```diff
@@ -1237,12 +1232,31 @@ \section{Discussion}
 used a heuristic algorithm to produce ``plausible'' ARGs
 (i.e., those that explained the data with few mutations and recombinations),
 and searched for associations with traits in the resulting ensemble of ARGs.
+(See \citet{wong2024general} for more historical discussion.)
 Our approach for extending haplotypes follows the same logic,
 that an ARG with fewer recombination events is more parsimonious,
 and thus more likely.
 For this reason, it will occasionally be wrong even if the trees are correct,
 although in practice this source of error is likely much smaller than error in tree inference itself.
 
+\paragraph{IBD in ARGs}
+The term ``identity by descent'' (IBD) is used to mean many different (but related) things,
+and length distributions of shared IBD segments can be used for inference of recent demographic history
+\citep[for instance,][]{alasadi2018estimating,ringbauer2017inferring,browning2015accurate}.
+A commonly-used definition in the context of a given ARG says that
+the two genomes share an IBD segment if each has inherited the segment from their common ancestor
+along a single path \citep{ralph2013geography}.
+Largely for computational reasons, this is the definition that is used in \tskit's
+IBD-finding methods 
+(by G.~Tsambos \citet{tskit}; see also \citet{tsambos2023link}).
+However, many simulation and inference methods produce ARGs
+as shown in \figref[A\&B]{fig:conceptual},
+in which inheritance of a single segment is represented by more than one edge.
+This means that the \texttt{ibd\_segments} method of \tskit{} will return shorter segments than it ought to.
+However, applying our method of extending haplotypes
+will remove this issue, and so should make returned IBD segments
+more closely match those predicted by theory.
```

Comment:
```suggestion
However, as we have shown above,
our method of extending haplotypes will modify the tree sequence
so that the inherited segment is represented by a single edge
(as in \figref[C\&D]{fig:conceptual}).
So, if our method (\texttt{extend_haplotypes}) is applied before finding IBD segments
(with  \texttt{ibd\_segments}),
then the resulting segments should much more accurately represent
the IBD segments (in the ``path'' sense used here) implied by the tree sequence.
(Whether the resulting segments better match those predicted by theory depends
also on the quality of tree inference.)
```

Reply:
This reads clearly to me.

Reply:
That looks good to me.